test-integration/test-config/tests/auto_airdrop_feepayer.rs

```diff
@@ -34,7 +35,10 @@ fn test_auto_airdrop_feepayer_balance_after_tx() {
             ..Default::default()
         },
         ledger: LedgerConfig {
-            resume_strategy: LedgerResumeStrategy::Reset,
+            resume_strategy_config: LedgerResumeStrategyConfig {
+                variant: LedgerResumeStrategyType::Reset,
+                reset_slot: Some(0),
```

Comment:
`Some(0)` shuold be same as `None` right since we default to `0` if not provided?